notebooks/I3C-Protocol-API/I3C-Protocol-Controller-API/I3C-Protocol-Controller-API-Basic/I3C-Protocol-Supernova-API.ipynb

```diff
@@ -1363,11 +1468,11 @@
     }
    ],
    "source": [
-    "TARGET_ADDR     = 0x08\n",
+    "TARGET_ADDR     = 0x0A      # LSM6DSV dynamic address\n",
```

Comment:
Use a variable to indicate the address, in this case corresponds to the `LSMD6V_DYNAMIC_ADDRESS`